streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamTask.java

```diff
@@ -320,7 +318,7 @@ private void commitOffsets(final boolean startNewTransaction) {
                 try {
                     consumer.commitSync(consumedOffsetsAndMetadata);
                 } catch (final CommitFailedException e) {
-                    log.warn("{} Failed offset commits {}: ", logPrefix, consumedOffsetsAndMetadata, e);
+                    log.warn("{} Failed offset commits {} due to CommitFailedException", logPrefix, consumedOffsetsAndMetadata);
```

Comment:
Nit: missing `:` (as discussed, it's subjective -- just put the comment as it is not consistent with the other log messages atm)